client/src/components/application/Transactions.vue

```diff
@@ -62,7 +62,7 @@
         </template>
 
         <v-layout id="transaction-title-bar" text-xs-center>
-          <v-flex class="py-2 fs-18 fw-700">TRANSACTION HISTORY</v-flex>
+          <v-flex class="py-2 fs-16 fw-700">TRANSACTION HISTORY</v-flex>
```

Comment:
Super minor but will cause a conflict. Maybe just easier to leave this out of your pr?

Reply:
Done. Please remember to make this change!

Also, lmk when your PR is merged and I can build/deploy this project to Dev.

Reply:
Thanks, absolutely, will do 👍